nextflow.config

```
@@ -9,13 +9,53 @@
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
-    // References
-    genome                     = null
-    igenomes_base              = 's3://ngi-igenomes/igenomes/'
-    igenomes_ignore            = false
+    transcript_fasta           = null
+
+    // Workflow options
+    QC_only                    = false
```

Parameter names should be all `snake_case`.

```suggestion
    qc_only                    = false
```